src/commands/deploy/latest.js

```diff
@@ -711,6 +713,14 @@ function handleCreateDeployError(output, error) {
     );
     return 1;
   }
```

Comment:
Add an empty line after this

Reply:
@leo On all the if-statements in `handleCreateDeployError`?

Reply:
Oh, I see. Then keep it!